vllm/distributed/kv_transfer/kv_connector/v1/nixl_connector.py

```diff
@@ -236,11 +236,11 @@ def add_new_req(
         _req = ReqMeta(
             local_block_ids=local_block_ids,
             local_physical_block_ids=local_block_ids,
-            remote_block_ids=kv_transfer_params["remote_block_ids"],
-            remote_engine_id=kv_transfer_params["remote_engine_id"],
-            remote_request_id=kv_transfer_params["remote_request_id"],
-            remote_host=kv_transfer_params["remote_host"],
-            remote_port=kv_transfer_params["remote_port"],
+            remote_block_ids=kv_transfer_params.get("remote_block_ids"),  # type: ignore
+            remote_engine_id=kv_transfer_params.get("remote_engine_id"),  # type: ignore
+            remote_request_id=kv_transfer_params.get("remote_request_id"),  # type: ignore
+            remote_host=kv_transfer_params.get("remote_host"),  # type: ignore
+            remote_port=kv_transfer_params.get("remote_port"),  # type: ignore
```

Comment:
@markmc , do you think this works?
I tried updating ReqMeta with optional, then the mypy check exploded. => Since we only want to align with toy_proxy, so I am hoping to use `# type ignore` , in that case, we can still use None and make mypy happy